source/server/README.md

```diff
@@ -143,7 +140,7 @@ normally) or 'validate' (validate configs and exit).
 Hot restart parent shutdown time in seconds
 
 --drain-time-s <uint32_t>
-Hot restart drain time in seconds
+Hot restart and LDS removal drain time in seconds
```

Comment:
Nit: prefer non-update changes to be distinct PRs.

Reply:
Our fix format flagged this, and this belongs to the associated envoy update as our test server CLI changed.. so I thought I'd include it here